test/distributed/fsdp/test_wrap.py

```diff
@@ -475,16 +483,16 @@ def test_auto_wrap_preset_force_leaf_custom(self):
         self.assertTrue(isinstance(model.module[1], nn.ModuleList))
 
     @unittest.skipIf(not torch.cuda.is_available(), "Test Requires CUDA")
-    @parametrize("fsdp_init_mode", [FSDPInitMode.CUDA_BEFORE, FSDPInitMode.CUDA_AFTER])
+    @parametrize("cuda_init_mode", [CUDAInitMode.CUDA_BEFORE, CUDAInitMode.CUDA_AFTER])
```

Comment:
subTest?

Reply:
Sounds good. Subtesting will be done in follow ups.